incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/config/MqttKafkaBindingConfig.java

```diff
@@ -45,4 +50,19 @@ public MqttKafkaRouteConfig resolve(
             .findFirst()
             .orElse(null);
     }
+
+    public String messagesTopic()
+    {
+        return options.topics.messages.asString();
+    }
+
+    public String sessionsTopic()
+    {
+        return options.topics.sessions.asString();
+    }
+
+    public String retainedTopic()
+    {
+        return options.topics.retained.asString();
+    }
```

Comment:
Please return `StringFW` and avoid the call to `.asString()` here.

If we keep the return `String`, then we need to re-convert `String` back to `StringFW` for each usage, meaning that we are UTF-8 encoding the same string repeatedly, instead of just once during configuration parsing.